### PR TITLE
LPS-118695

### PR DESCRIPTION
**Pull request by brunoquei2105**

### CODE REVIEW

**Comment by brunoquei2105** [comment]
ci:test:sf

**Comment by brunoquei2105** [comment]
ci:test:relevant

**Review by drewbrokke — CHANGES_REQUESTED** [review]
@brunoquei2105 Nice work on this. You're definitely on the right track.

This change is good, but there's actually another view that has the same validation on it for a user's screen name.

Navigate to Users & Organizations and click the '+' button to see it in action.

In the end, you'll essentially want to copy what is done on that page, as it automatically sets the proper `maxlength` on the field itself.

**Comment by drewbrokke** [comment]
ci:test:sf

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/add_account_user.jsp [hunk 1]
```diff
@@ -82,7 +85,20 @@ renderResponse.setTitle(LanguageUtil.format(request, "add-new-user-to-x", accoun
...
-					<aui:input label="screen-name" name="screenName" required="<%= true %>" type="text" />
+					<aui:input label="screen-name" name="screenName" type="text">
+
+						<%
+						ScreenNameValidator screenNameValidator = ScreenNameValidatorFactory.getInstance();
+						%>
+
+						<c:if test="<%= Validator.isNotNull(screenNameValidator.getAUIValidatorJS()) %>">
+							<aui:validator name="required" />
+							<aui:validator name="maxLength">75</aui:validator>
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/add_account_user.jsp:96]: You can remove these after adding the `aui:model-context` tag

File: modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/add_account_user.jsp [hunk 2]
```diff
@@ -82,7 +85,20 @@ renderResponse.setTitle(LanguageUtil.format(request, "add-new-user-to-x", accoun
 						<liferay-ui:message key="<%= usne.screenNameValidator.getDescription(locale) %>" />
 					</liferay-ui:error>
 
-					<aui:input label="screen-name" name="screenName" required="<%= true %>" type="text" />
+					<aui:input label="screen-name" name="screenName" type="text">
```
Comment (drewbrokke) [inline modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/add_account_user.jsp:88]: You can remove the `label` and `type` attributes after adding the `aui:model-context` tag.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
See my inline comments. Before pushing the changes, make sure to do two things:
- Run Source Formatter locally and fix any issues.
- Put the changes into a single commit
Thanks!

**Comment by drewbrokke** [comment]
@brunoquei2105 code looks great.

I'd fix the commit message in 2 ways:
1. Remove the hyphen after the ticket number.
2. Remove the body lines (the ones after the first line). Usually a body is included to provide further explanation or rationale for a change. In this case I think you can use just the one-line commit message.

**Comment by drewbrokke** [comment]
LGTM 👍

**Review by drewbrokke — APPROVED** [review]
LGTM 👍

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**